### DISCUSSION TITLE
quarto-giscus/about

### DISCUSSION THREAD

**Discussion by giscus** [r-leyshon/quarto-giscus#4] [Announcements]
# quarto-giscus/about

https://r-leyshon.github.io/quarto-giscus/about.html

**Comment by r-leyshon** [comment]
testing

**Comment by r-leyshon** [comment]
# Testing

~~markdown~~ **functionality**

[some link](https://www.google.com.mx/)

![some image](https://i.imgur.com/MiCmAtN.jpeg)
